### DISCUSSION TITLE
No Game Connection to Dirt Rally 2.0

### DISCUSSION THREAD

**Discussion by Scipio117** [cosmii02/RacingDSX#45] [Q&A] [answered]
Pretty much the title, i've already changed the hardware_settings_config file but there is no connection to the game. Any idea what it the problem may be???

![Capture](https://github.com/cosmii02/RacingDSX/assets/133292136/5991ba35-aa1d-4df8-b16f-cd762ab0f82c)

**Comment by jansteffen** [comment, marked as answer]
I have a few ideas:

1. For some reason the executable for Dirt Rally 2.0 is not included by default in "Executables to Match"In the _Misc_ Tab. To manually add it, click _Add_ on the right, enter `dirtrally2` and click OK, then Apply.
2. Double check that the specified Port in `hardware_settings_config` matches the Game Port in RacingDSX
3. If all that fails maybe a firewall is blocking some of the UDP sending/listening

**Reply by Scipio117** [reply]
Thanks a lot, adding the executable fixed it.

**Reply by cosmii02** [reply]
should be in there by default in 0.6.1 now
